### DISCUSSION TITLE
Background image not displaying on Github pages site

### DISCUSSION THREAD

**Discussion by ThaboMtetwa** [community/community#56856] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

I deployed a page to Github pages and it displays everything except the background image. Im really wracking my brain to try figure out what is going on. Please help.

Here is the repo. The url for the background image is in the CSS file:

https://github.com/ThaboMtetwa/Chantells-Bakery.git

I am having this issue with multiple sites that I am trying to deploy so a fix for this will help me greatly.

**Comment by mrepol742** [comment, marked as answer]
Try this
instead of
``` css
    background-image: url('../Chantelles-Bakery/ales-krivec-QnNqGoCnBg0-unsplash.jpg');
```
use this

``` css
    background-image: url('ales-krivec-QnNqGoCnBg0-unsplash.jpg');
```

**Reply by ThaboMtetwa** [reply]
This worked thank you
